ArduPlane/Plane.h

```diff
@@ -385,8 +385,10 @@ class Plane : public AP_HAL::HAL::Callbacks {
     AP_BattMonitor battery;
 
 #if FRSKY_TELEM_ENABLED == ENABLED
+    // Inertial Navigation
+    AP_InertialNav_NavEKF inertial_nav {ahrs};
```

Comment:
No, InertialNav is only used by Copter and there has been talks about removing it actually.